doc/WebSite/Dapper-Usage.html

```diff
@@ -0,0 +1,187 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+	<head>
+		<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+		<title>Dapper Usage</title>
+		<link type="text/css" rel="stylesheet" href="bootstrap.min.css" />
+	</head>
+
+	<body>
+
+		<div class="document-contents">
+
+
+		<h3 id="DocIntroduction">Introduction</h3>
+			<p><a href="https://github.com/StackExchange/Dapper" target="_blank">Dapper</a> is an object-relational mapping (ORM) product for .NET. 
+			It provides a framework for mapping an object-oriented domain model to a traditional relational database. 
+			Dapper is free as open source software that is distributed under dual license, either the Apache License 2.0 or the MIT License.
+			<a href="https://www.nuget.org/packages/Abp.Dapper" target="_blank">Abp.Dapper</a> package simply integrates Dapper to ASP.NET Boilerplate.</p>
```

Comment:
No need to go details of licensing. "It provides a framework for mapping an object-oriented domain model to a traditional relational database." is also unnecessary since ORM explaines it.